jekyll/_cci2/faq.md

```diff
@@ -259,8 +255,7 @@ For the vast majority of customers, you can keep your current plan for now and t
 
 #### What are credits?
 {:.no_toc}
-Credits are used to pay for your usage based on machine type and size. Credits
-can also be used to pay for features, such as Docker Layer Caching.
+Credits are used to pay for users and usage. Credits can also be used to pay for features, such as Docker Layer Caching.
```

Comment:
Suggestion:
Credits are used to pay for users and usage based on machine type, size, and features such as Docker Layer Caching.

Reply:
Fixed